collection-scripts/gather_nodes

```diff
@@ -34,6 +34,7 @@ do
     mkdir -p ${NODE_PATH}
     oc exec -it $pod -n node-gather -- ip a >> $NODE_PATH/ip.txt
     oc exec -it $pod -n node-gather -- ip -o link show type bridge >> $NODE_PATH/bridge
+    oc exec -it $pod -n node-gather -- bridge vlan show >> $NODE_PATH/vlan
```

Comment:
Better dump it as json so it's easier to consume by tooling.

```bash
bridge -j vlan show
```

Reply:
Done